inst/resources/pyproject.py

```diff
@@ -0,0 +1,166 @@
+"""
+Support for detecting various information from python projects metadata.
```

Comment:
File ported over from `rsconnect-python` with minor adjustments for `typing` notations